docs/pages/blog/mui-product-comparison.md

```diff
@@ -0,0 +1,164 @@
+---
+title: An introduction to the MUI ecosystem
+description: MUI is more than just Material UI. Consider Joy UI, MUI Base, MUI X, and Toolpad for your next project.
+date: 2022-10-13T00:00:00.000Z
+authors: ['samuelsycamore']
+card: false
+tags: ['Product']
+---
+
+MUI is so much more than Material UI!
+
+You'll be forgiven if you thought MUI was merely shorthand for our most popular product.
+What began as an open-source React UI library has evolved over time into a full-fledged startup with a whole suite of related products.
+
+Material UI is the flagship; but it's also the gateway to MUI's ever-expanding ecosystem of UI tools.
+
+Though our roots are in [Material Design](https://material.io/), we're branching out well beyond those constraints these days to deliver a wider range of tools for developers to ship new features faster.
+
+Our primary offerings fall into two product lines: Core and X.
+MUI Core contains our foundational component libraries (like Material UI), while MUI X offers components that are significantly more complex (like the Data Grid).
+
+We're also in the early stages of developing a low-code internal tool builder called [MUI Toolpad](https://mui.com/toolpad/), in which you can build with every Core and X component in a drag-and-drop interface.
+
+Read on for more details on each of our products.
+
+## MUI Core
+
+The Core is MUI's foundational product line.
+It grew out of Material UI, and that library's legacy lives on in the name of the repo on GitHub: [`mui/material-ui`](https://github.com/mui/material-ui).
+
+But this repo contains much more than just Material UI these days.
```

Comment:
I'm not sold on the use of these adverbs. When I see "much, "lot", "just", or equivalent then it quickly becomes obvious where the person is trying to orient the discussion. It often feels like putting his own judgment above how facts could resonate with the people the message is written for. This get me to this alternative:

```suggestion
But this repository now contains more than Material UI.
```

We could also maybe expand on the product direction taken, I think that it's as simple as:

```suggestion
But this repository now contains more than Material UI. We have disassembled Material UI to expose its best parts as isolated open-source projects.
```